# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../udev/hotfix-rules2.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From: Kay Sievers <kay.sievers@vrfy.org>
Date: Sat, 25 Aug 2007 12:07:02 +0000 (+0200)
Subject: rules: random and urandom are 0666
X-Git-Url: http://git.kernel.org/?p=linux%2Fhotplug%2Fudev.git;a=commitdiff_plain;h=b6e4b08dc14444790c1560108e93a67e57e1148b

rules: random and urandom are 0666

Thanks to: Alexander E. Patrakov <patrakov@ums.usu.ru>.
---

diff --git a/etc/udev/rules.d/50-udev-default.rules b/etc/udev/rules.d/50-udev-default.rules
index b3c0cfc..f869f51 100644
--- a/etc/udev/rules.d/50-udev-default.rules
+++ b/etc/udev/rules.d/50-udev-default.rules
@@ -17,18 +17,16 @@ KERNEL=="hvc*|hvsi*",		GROUP="uucp"
 KERNEL=="lirc0",		SYMLINK+="lirc"
 
 # mem
-KERNEL=="null|zero|random",	MODE="0666"
+KERNEL=="null|zero|full|random|urandom", MODE="0666"
 KERNEL=="null",			SYMLINK+="XOR"
 KERNEL=="mem|kmem|port",	GROUP="kmem", MODE="0640"
 KERNEL=="nvram",		GROUP="kmem", MODE="0600"
 KERNEL=="ram0",			SYMLINK+="ramdisk"
 KERNEL=="ram1",			SYMLINK+="ram"
-KERNEL=="urandom",		MODE="0644"
-KERNEL=="full",			MODE="0666"
 
 # input
 KERNEL=="mouse*|mice|event*",	NAME="input/%k", MODE="0640"
-KERNEL=="ts[0-9]*|uinput",		NAME="input/%k", MODE="0600"
+KERNEL=="ts[0-9]*|uinput",	NAME="input/%k", MODE="0600"
 KERNEL=="js[0-9]*",		NAME="input/%k", MODE="0644", SYMLINK+="%k"
 
 # video4linux
